storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="ephemeral-storage-csi-vol-detach-non-graceful-shutdown"]
+= CSI driver volume detach with non-graceful node shutdown
+include::_attributes/common-attributes.adoc[]
+:context: ephemeral-storage-csi-vol-detach-non-graceful-shutdown
+
+toc::[]
+
+This feature allows Container Storage Interface (CSI) drivers to automatically detach volumes when a node goes down non-gracefully.
+
+:FeatureName: CSI driver volume detach with non-graceful node shutdown
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+This feature is enabled by default.
```

Comment:
It can be confusing to say that the feature is TP and enabled by default

Reply:
@gcharot Normally TP features need enabling from the user to, such as through feature gates. However, this feature has a feature gate, but it is enabled by default, so I'm not sure what to say that is less confusing.

Reply:
I've removed the "enabled" note.